plugins/sinks/wavefront/wavefront.go

```diff
@@ -152,7 +152,7 @@ func (sink *wavefrontSink) sendPoint(metricName string, value float64, timestamp
 	if len(sink.Prefix) > 0 {
 		metricName = sink.Prefix + "." + metricName
 	}
-	if sink.filters != nil && !sink.filters.Match(metricName, tags) {
+	if sink.filters != nil && !sink.filters.MatchMetric(metricName, tags) {
```

Comment:
We need to add logic to filter tags since this was likely relaying on `MatchMetrics`'s behavior